src/elements/content-sidebar/SidebarPanels.js

```diff
@@ -96,31 +103,45 @@ const SidebarPanels = ({
                 enabled={hasActivityFeed}
                 path={`/${SIDEBAR_VIEW_ACTIVITY}`}
                 pathFallback={`/${SIDEBAR_VIEW_DETAILS}`}
-                render={() =>
+                render={({ match }) =>
                     isOpen && (
-                        <LoadableActivitySidebar
-                            currentUser={currentUser}
-                            file={file}
-                            onVersionHistoryClick={onVersionHistoryClick}
-                            {...activitySidebarProps}
-                            startMarkName={MARK_NAME_JS_LOADING_ACTIVITY}
-                        />
+                        <Switch>
+                            <Route exact path={match.path}>
+                                <LoadableActivitySidebar
+                                    currentUser={currentUser}
+                                    file={file}
+                                    onVersionHistoryClick={onVersionHistoryClick}
+                                    startMarkName={MARK_NAME_JS_LOADING_ACTIVITY}
+                                    {...activitySidebarProps}
+                                />
+                            </Route>
+                            <Route exact path={`${match.path}/versions`}>
+                                <LoadableVersionsSidebar fileId={fileId} key={fileId} />
```

Comment:
should we pass in version sidebar props even if we don't use any real data yet?

Reply:
I don't understand the question. The version sidebar doesn't really support much in the way of props at the moment.

Reply:
we're doing it for the other sidebars `{...activitySidebarProps}` but it makes sense to wait until we actually do something with said props.